sdks/python/apache_beam/dataframe/transforms_test.py

```diff
@@ -17,17 +17,61 @@
 from __future__ import absolute_import
 from __future__ import division
 
+import typing
 import unittest
 
 import pandas as pd
+from past.builtins import unicode
 
 import apache_beam as beam
+from apache_beam import coders
 from apache_beam.dataframe import expressions
 from apache_beam.dataframe import frame_base
 from apache_beam.dataframe import transforms
 from apache_beam.testing.util import assert_that
 
 
+def sort_by_value_and_drop_index(df):
+  if isinstance(df, pd.DataFrame):
+    sorted_df = df.sort_values(by=list(df.columns))
+  else:
+    sorted_df = df.sort_values()
+  return sorted_df.reset_index(drop=True)
```

Comment:
Note there's actually a diff here from the original `check_correct`. It sorts by value and resets the index rather than sorting by index.  I had to do this because the concatenated indices of the batches (e.g. `[0,1,2,0,1,0,]`) wouldn't match the index in my expected df (e.g. `[0,1,2,3,4,5]`).